src/Bigcommerce/Api/Client.php

```diff
@@ -258,7 +258,19 @@ public static function setConnection(Connection $connection = null)
      */
     public static function getCollection($path, $resource = 'Resource')
     {
-        $response = self::connection()->get(self::$api_path . $path);
+        $apiVersion = 2;
+        $api_path = self::$api_path;
+
+        if (strpos($path, '/customers/subscribers') !== false) {
+            $apiVersion = 3;
+            $api_path = str_replace('v2', 'v3', $api_path);   
+        }
+
+        $response = self::connection()->get($api_path . $path);
+
+        if ($apiVersion == 3) {
```

Comment:
Let's do a strict type-comparison here, and add a comment explaining why this is necessary